### ISSUE TITLE
Redirect link for Project cancelation  in notification list 

### ISSUE THREAD

**Issue by maryjaf** [Giveth/giveth-dapps-v2#3058]
When My project is cancelled by admin user I receive a notification like below:
Your project [test999999](https://giveth.io//project/test999999) has been canceled by an admin. More information can be found in our [terms of service](https://giveth.io/tos) .
by tapping on name of project I see this message ,it's better the showing message in this page is revised , or the name of project is not clickable, what do you think? @divine-comedian 
<img width="1431" alt="image" src="https://github.com/Giveth/giveth-dapps-v2/assets/111529185/c70f6e4f-7af2-4ab7-98d5-19afc7935aae">

**Comment by divine-comedian** [comment]
Yep good point! Maybe we can add to this text and provide a link for the user 

`Oops! This project is no longer available or not found! If you think this is a mistake reach out to us on Discord`[link to discord.giveth.io]`"

**Comment by MoeNick** [comment]
@divine-comedian This is a general error that prevents users from seeing the canceled projects page, I suggest we won't make the link in the notification message if the notif is for canceled projects.

**Comment by MoeNick** [comment]
- [] Removing the link to the project, if it's a canceled project notification. 
- [] Editing the copy of canceled project single view.

**Comment by MoeNick** [comment]
Cool, based on @maryjaf 's comment I edited the scope:
- [ ] Removing the link to the project, if it's a canceled project notification. 
- [ ] Editing the copy of the canceled project single, if the owner opens his/her own project. 

As we discussed with @RamRamez and @mohammadranjbarz ,  1st one is a backend issue

**Comment by maryjaf** [comment]
project owner view
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaTVtIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--e9ead17c86d3c3801a8927f3ac88f290c81d2fad/image.png)

Project view:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBakptIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--898aa34d2b9fc3394ab480fc0a3e1ba07560ab95/image.png)
Is that ok? @divine-comedian

**Comment by divine-comedian** [comment]
@RamRamez - can we remove the toast notification? I think it's confusing

**Comment by RamRamez** [comment]
> @RamRamez - can we remove the toast notification? I think it's confusing

@maryjaf It's on staging now and you can test it.
